### PR TITLE
Game Play

### PR DESCRIPTION
**Pull request by dehli**
This change does a bit of styling updates to the game-play view. As you can see from the code, it's not all hooked up yet (I'll need to do a little more backend stuff to add all the needed functionality).

A lot of these updates won't result in any changes until the backend changes are also merged (but the code won't break if it's merged before the backend is).

The chat is scrollable now. I need to add auto scroll to the bottom whenever a new message is received. I also add the various titles to the page.

### CODE REVIEW

File: less/panels/game-play.less [hunk 1]
```diff
@@ -1,27 +1,75 @@
...
+                        position: absolute;
+                        border-bottom: 4px solid @wombat-green;
+                        left: 0;
+                        bottom: -11px;
+                    }
+                }
+            }
+
+            .chat-box {
+                height: calc(~"100% - 40px");
```
Comment (oconn) [inline less/panels/game-play.less]: Where does this 40px come from? Is it possible to put it in a variable or is it really arbitrary?

File: src/cljs/wombats_web_client/panels/game-play.cljs [hunk 2]
```diff
@@ -36,19 +36,56 @@
...
+  (let [{:keys [status
+                start-time]} @info]
+    [:h1.game-play-title 
+     (case status
+       :closed
+       "GAME OVER"
+           
+       (:pending-open
+        :pending-closed)
+       "ROUND 1 STARTS IN: 2:06"
```
Comment (oconn) [inline src/cljs/wombats_web_client/panels/game-play.cljs]: Is there a ticket to support this feature and the `:active` case? If so could you add a TODO comment with the ticket #?

Comment (dehli) [inline src/cljs/wombats_web_client/panels/game-play.cljs]: So I was going to keep it with the same ticket (#21). I'll add a comment referencing it and update the ticket.

File: less/panels/game-play.less (hunk 1 above)
Comment (dehli) [inline less/panels/game-play.less]: Good catch! It comes from the `.chat-title` and there was actually an inconsistency so switching to a variable fixed that 👍

**Pull request merged**